src/main/java/Deadlines.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Deadlines tasks that need to be done before a specific date/time
+ * e.g., submit report by 11/10/2019 5pm
+ */
+
```

Comment:
Consider, perhaps, deleting this line break between the comments and the function for consistency